src/python/bcc/containers.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Kinvolk GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def filter_by_containers(bpf_text, args):
+    bpf_text = bpf_text.replace('CONTAINERS_FILTER_HEADER',
+        """
+            #if CGROUP_ID_SET
```

Comment:
What about using `#ifdef`? If so, then you **don't** have to assign it to `0` in the negative case.